packages/app-webdir-ui/src/helpers/dataConverter.js

```diff
@@ -277,7 +277,7 @@ export const staffConverter = ({
 
   // We use EID if it's available, otherwise we use the asurite_id.
   const profileURLBase = options.profileURLBase ?? "";
-  const asuriteEID = filledDatum.eid.raw
+  const asuriteEID = filledDatum.eid.raw !== '0'
```

Comment:
@tbutterf Couple issues here. 
1. It's only applying to the `staffConverter` and should also be implemented for the `studentsConverter`
2. You're regressing the logic on line 280 by replacing the original check with a different check. `filledDatum.eid.raw && filledDatum.eid.raw !== "0"` (untested) would be more appropriate.

Reply:
Ah. Dang. Thanks. I'll update it.

Reply:
I have updated it according to your notes @mlsamuelson. I didn't add the question marks that @davidornelas11 suggested, but I can, if needed.

Reply:
The data could change at some point, and I always appreciate code that's guarded against a rainy day scenario, but two mitigating circumstances on this:
1. We've been okay thus far, and checking the returns from our API, looks like if the field is returned, raw is always included, and Workday is a way off and will involve testing before we roll out changes.
2. We're about to put the data converters under the microscope with the code review, so there may be changes coming to this area, and/or we can identify this change as an outcome of that effort. I'm guessing there are more opportunities to address the raw issue throughout the data converters, and if we make that DRY, we can tackle all at once, perhaps.

Thus I'm not going to have this hung up on that detail. :)